packages.dot

```diff
@@ -1,5 +1,6 @@
 digraph packages {
 	"/service/cluster" -> "/service/k8sclient";
+	"/service/cluster" -> "/service/k8sclient/kubectl";
```

Comment:
May we avoid that dependency on the implementation details?

Reply:
See https://github.com/percona-platform/dbaas-controller/pull/137

Reply:
:+1: I removed the dependency.